src/shared/lib/MutationUtils.ts

```diff
@@ -232,7 +232,18 @@ export function extractGenomicLocation(mutation: Mutation)
 // TODO remove when done refactoring mutation mapper
 export function genomicLocationString(genomicLocation: GenomicLocation)
 {
-    return `${genomicLocation.chromosome},${genomicLocation.start},${genomicLocation.end},${genomicLocation.referenceAllele},${genomicLocation.variantAllele}`;
+    // mapping chromosome X with 23
+    let chromosome = "";
+    if (genomicLocation.chromosome === "X") {
```

Comment:
you should use a switch statement instead of if/else when testing against a single value (genomicLocation.chromosome).   Good to get in habit of using switch in these kinds of situations.  You could also use a an enum with a default.  If genomicLocation.chromosome exists in enum, you use it's value.  If not, you use Enum.default.